yoti-sdk-impl/src/test/java/com/yoti/api/client/spi/remote/call/RemoteProfileServiceTest.java

```diff
@@ -32,6 +32,7 @@
 
 public class RemoteProfileServiceTest {
     private static final String MESSAGE_PREFIX = "GET&";
+    private static final String YOTI_SDK_HEADER = "X-Yoti-SDK";
```

Comment:
Aren't we calling it SDK identifier in the other SDKs?

Reply:
I just used the same name we have in RemoteProfileService.java, where we use:
```Java
private static final String AUTH_KEY_HEADER = "X-Yoti-Auth-Key";
private static final String DIGEST_HEADER = "X-Yoti-Auth-Digest";
private static final String YOTI_SDK_HEADER = "X-Yoti-SDK";
````

Reply:
Ok